rclpy/src/rclpy_common/src/common.c

```diff
@@ -109,6 +117,31 @@ rclpy_convert_to_py_names_and_types(rcl_names_and_types_t * names_and_types)
   return pynames_and_types;
 }
 
+static
+PyObject *
+_convert_rmw_time_to_py_duration(rmw_time_t * duration)
+{
+  uint64_t total_nanoseconds = RCUTILS_S_TO_NS(duration->sec) + duration->nsec;
+
+  PyObject * pyduration_module = PyImport_ImportModule("rclpy.duration");
+  if (!pyduration_module) {
+    return NULL;
+  }
+  PyObject * pyduration_class = PyObject_GetAttrString(pyduration_module, "Duration");
+  Py_DECREF(pyduration_module);
+  if (!pyduration_class) {
+    return NULL;
+  }
+
+  PyObject * args = PyTuple_New(0);
```

Comment:
Need to check if `args` is `NULL`, and if so `Py_DECREF(pyduration_class)` and return `NULL`

Reply:
Need to check if `args` is null, `PyObject_Call` [forbids it being  from `NULL`](https://github.com/python/cpython/blob/d6bf6f2d0c83f0c64ce86e7b9340278627798090/Include/object.h#L127)